packages/cli/src/controllers/debug.controller.ts

```diff
@@ -18,17 +19,26 @@ export class DebugController {
 	async getMultiMainSetupDetails() {
 		const leaderKey = await this.multiMainSetup.fetchLeaderKey();
 
-		const activeWorkflows = await this.workflowRepository.find({
+		const triggersAndPollers = await this.workflowRepository.find({
```

Comment:
This is very interesting but a gentle reminder that it would return the current main's information, so if not the leader, this should return empty.

Reply:
Exactly, I'll use this for debugging, to check for unexpected follower state. Added `isLeader` to make this more explicit.